site/docs-md/apis/local-notifications/index.md

```diff
@@ -33,7 +33,9 @@ LocalNotifications.schedule({
       sound: null,
       attachments: null,
       actionTypeId: "",
-      extra: null
+      extra: null,
+      smallIcon: null,
+      iconColor: null
```

Comment:
since they are optional, don't add them to the sample, or if you add them, do it with a real value and not null.
In this case, null crash the app (see my previous try catch comment)

Reply:
is there a better way to show users that these values are optional anywhere in docs?  Feel like leaving it off doesn't let people know that they are there.

Reply:
Most people will use typescript, so they can easily check the types.

Docs are generated from the types, but don't work very well in some cases, so checking the types directly is better than the website docs.

But as I said, if you want to change the example to use smallIcon and iconColor, just use valid values instead of null

Reply:
removed for consistency